app/controllers/api/v1/messages_controller.rb

```diff
@@ -7,7 +7,7 @@ class MessagesController < Api::V1::BaseController
       MESSAGES_LIMIT = 10
 
       def index
-        @messages = room.messages.includes(:user).order(id: :desc).limit(MESSAGES_LIMIT)
+        @messages = room.messages.kept.includes(:user).order(id: :desc).limit(MESSAGES_LIMIT)
```

Comment:
We should display all messages but body of discarded ones should be replaced by '[message has been deleted]'